utils/readme/src/readmes/rc-tooltip.tsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+
+import {
+  DocProps,
+  Docs,
+  DocsExample,
+  DocsP,
+  DocsSubtitle
+} from '@rmwc/doc-utils';
+import examples from '../generated-examples/rc-tooltip.json';
+import propsSrc from '../generated-props/rc-tooltip.json';
+
+import { RCTooltip } from '@rmwc/rc-tooltip';
+
+import { IconButton } from '@rmwc/icon-button';
+import { Button } from '@rmwc/button';
+import { Avatar } from '@rmwc/avatar';
+import { RMWCProvider } from '@rmwc/provider';
+
+export default function Readme() {
+  return (
+    <Docs
+      title="RC Tooltips"
+      lead="Tooltips display informative text when users hover over, focus on, or tap an element."
+      module="@rmwc/rc-tooltip"
+      styles={['@rmwc/rc-tooltip/tooltip.css']}
+      examples={examples}
+      addon
+    >
+      <DocsSubtitle>Basic Usage</DocsSubtitle>
```

Comment:
Should we add a section above here stating that this is not the official mdc tooltip?